lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -21,6 +23,23 @@
     'lti',
 ]
 
+# exclude these in Metrics
+non_student_roles = ['instructor', 'staff']
+
+def get_non_student_list(course_key):
+    # find all user_ids with instructor or staff roles in student_courseaccessrole table
+    non_student_idlist=[]
```

Comment:
Thanks for the tip.  I already ran 'paver run_quality' and fixed all the styling problems!   
Next time my code won't look like from someone who is a newbie to Python :)